examples/features/advancedtls/README.md

```diff
@@ -6,7 +6,7 @@ The servers run a basic echo server with the following setups:
 * Port 8884: A server with a revoked certificate using certificate providers and crl providers.
 * Port 8883: A server running using InsecureCredentials.
 
-The clients are designed to call these servers with varying configurations of credentials and revocation configurations.
+Clients are designed to call these servers with varying configurations of credentials and revocation configurations.
```

Comment:
why change this? The clients seems fine here

Reply:
These got changed by mistake, changing them back now.